README.md

```diff
@@ -13,50 +13,120 @@
   </p>
 </div>
 
+## Release notes
+**Release notes and unreleased changes can be found in the [CHANGELOG](CHANGELOG.md)**
+
 ## Requirements
 - rust v1.56 and up
-- MacOS (x86), M1 (through rosetta) or Linux (x86) for sandbox tests. Testnet is available regardless
+- MacOS (x86) or Linux (x86) for sandbox tests. Testnet is available regardless
+
+### M1 MacOS
+NOTE: Current version of workspaces does not provide support for M1 macbooks, due to some internal upgrades with wasmer. Look at using for workspaces version `0.1.1` for now until we get this settled out or check the progress of it with this [issue](https://github.com/near/workspaces-rs/issues/110).
 
-### M1 MacOS Setup
 To be able to use this library on an M1 Mac, we would need to setup rosetta plus our cross compile target:
 ```
 softwareupdate --install-rosetta
 rustup default stable-x86_64-apple-darwin
 ```
 
-## Testing
-A simple test to get us going and familiar with the features:
+### Note about compiling with contract
+`workspaces-rs` does not currently support compiling to WASM. So, if we are compiling this library alongside a `wasm32` target such as compiling alongside a NEAR contract, it is best if we put this dependency in `[dev-dependencies]` section to avoid any conflicts.
```

Comment:
Discussed in chat. But I initially read this as a reference to the contract code being complied rather than workspaces itself

Reply:
@doriancrutcher @AustinBaggio should look better now